speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/EndExperimentMenu.h

```diff
@@ -0,0 +1 @@
+void* EndExperimentInit();
```

Comment:
requires pragma once, otherwise it might cause linker errors later on 
```suggestion
#pragma once
extern void* EndExperimentInit();
```